src/Infrastructure/Repository/TaskYamlRepository.php

```diff
@@ -433,4 +434,51 @@ protected function buildTaskSet(array $taskData, array $taskListData, array $tas
 
         return $task;
     }
+
+    /**
+     * @FIXME
+     *
+     * - This code is needed to be moved to separate repo (TaskPhpRepository)
+     * - To think about $this->existingTasks as well. I think we don't need them here.
+     *
+     * @return array<string, \SprykerSdk\Sdk\Core\Domain\Entity\Task>
+     */
+    private function getPHPTasks(): array
```

Comment:
Please maintain a consistent naming. changePhpCommand <=> getPhpTasks.

Reply:
changePhpCommand looks like an adequate function, and my like a dirty hack. But ok, i rename it.